packages/rancher-sachet/charts/templates/_helpers.tpl

```diff
@@ -0,0 +1,92 @@
+{{- define "system_default_registry" -}}
+{{- if .Values.global.cattle.systemDefaultRegistry -}}
+{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Windows cluster will add default taint for linux nodes,
+add below linux tolerations to workloads could be scheduled to those linux nodes
+*/}}
+
+{{- define "linux-node-tolerations" -}}
+- key: "cattle.io/os"
+  value: "linux"
+  effect: "NoSchedule"
+  operator: "Equal"
+{{- end -}}
+
+{{- define "linux-node-selector" -}}
+{{- if semverCompare "<1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+beta.kubernetes.io/os: linux
+{{- else -}}
+kubernetes.io/os: linux
+{{- end -}}
+{{- end -}}
+
+{{/*
+Allow the release namespace to be overridden for multi-namespace deployments in combined charts
+*/}}
+{{- define "sachet.namespace" -}}
+  {{- if .Values.global.namespaceOverride -}}
+    {{- .Values.global.namespaceOverride -}}
+  {{- else -}}
+    {{- .Release.Namespace -}}
+  {{- end -}}
```

Comment:
should we use default here?

Reply:
One of @cbron 's comments says we should not use the default namespace.

Reply:
Sorry, I meant default as in the Helm function default. e.g.

```suggestion
{{ default .Release.Namespace .Values.global.namespaceOverride }}
```

Reply:
fixed.